scripts/upgrade

```diff
@@ -225,11 +237,26 @@ then
   ynh_secure_remove $backup_bdd
 fi
 
+if  [ -d "$backup_data" ]
+then
+  cp -ar $backup_data $final_path/data
+  ynh_secure_remove  $backup_data
+fi
+
+if  [ -d "$backup_skel_dist" ]
+then
+  cp -ar $backup_skel_dist $final_path/www/skel-dist
+  ynh_secure_remove  $backup_skel_dist
+fi
+
+
 #=================================================
 # ADD A CONFIGURATION
 #=================================================
 ynh_script_progression --message="Adding a configuration file..." --weight=5
 
+echo -e "\n /** \n * Activer la possibilité de faire une mise à jour semi-automatisée \n * depuis fossil.kd2.org. \n * \n * Si mis à TRUE, alors un bouton sera accessible depuis le menu "Configuration" \n * pour faire une mise à jour en deux clics. \n * \n * Il est conseillé de désactiver cette fonctionnalité si vous ne voulez pas \n * permettre à un utilisateur de casser l'installation ! \n * \n * Défaut : true \n * \n * @var bool \n */ \n \nconst ENABLE_UPGRADES = false;" >> $final_path/config.local.php
```

Comment:
If I am not wrong, the above line run twice adds the constant twice to `config.local.php`. So 2 upgrades will imply a warning after the restart of the app.

If I understand correctly, you don't overwrite the `config.local.php` file so the user may override the default config.

But here, you need to disable one functionality activated by default.

What about a `config.local.php` which would simply and only [require](https://www.php.net/manual/en/function.require.php) a `config.local.yunohost.php` (which you can overwrite during the installation and every upgrades then) and a `config.local.user.php` (which will be added once during the installation and left intact so the user can overwrite it)?

I can think of a solution for the existing installations if you want, so the upgrades of the existing packages would also work :).

Reply:
How to explain... From the beginning, the app to be installed need a copy from the file `config.dist.php` in a file `config.local.php`. Some lines are edited when fist installation. But with this release, there is new lines added with a new function where upgrades by default can be done from the app. It is not recommended in Yunohost environment accept this function. I want keep the file `config.local.php` but how add this new constant to disable upgrades ? 
I am not sure I can add a file as you say, or how to do this... Or can I use another way to add these new lines with a `diff` from the the original file `config.dist.php` ?

Reply:
Ok, I can imagine that the file `config.local.php` is just a copy of `config.dist.php` (all constants are commented in this file expecting `namespace Garradin;`
Then, I can keep the last `config.local.php` before upgrade renaming this file `config.local.yunohost.php`, call this file with a line `require 'config.local.yunohost.php'` and just add in this file a new line with `const ENABLE_UPGRADES = false;`.

I am trying debug a mistake when we have from few versions use a copy of `config.dist.php` on the install script and just need to write 3 lines on the file `config.local.php` with

```
# copy config.dist.php creating config.local.php
cp -a $final_path/config.dist.php $final_path/config.local.php
# create a key for the secret key CSFR;
key=$(ynh_string_random --length=50)
# edit config.local.php
ynh_replace_string --match_string="const SECRET_KEY = '3xUhIgGwuovRKOjVsVPQ5yUMfXUSIOX2GKzcebsz5OINrYC50r';" --replace_string="const SECRET_KEY = '$key';" --target_file="$final_path/config.local.php"
if [ "$path_url" == "/" ]; then
  ynh_replace_string --match_string="//const WWW_URI = '/asso/';" --replace_string="const WWW_URI = '$path_url';" --target_file="$final_path/config.local.php"
else
  ynh_replace_string --match_string="//const WWW_URI = '/asso/';" --replace_string="const WWW_URI = '$path_url/';" --target_file="$final_path/config.local.php"
fi
```
and now only need add

`ynh_replace_string --match_string="//const ENABLE_UPGRADES = true;" --replace_string="const ENABLE_UPGRADES = false;" --target_file="$final_path/config.local.php"`

I can also just keep the file `config.local.php` on the upgrade script and just add the line `const ENABLE_UPGRADES = false;` at the end with `echo` ??

What is better ? @alexAubin encourage me `_not_ remove the `--keep` in `ynh_setup_source` 
Maybe I keep the file `config.local.php` and just add inside as you say a `require 'config.local.yunohost.php'` with the new lines to add...

Reply:
I try add a file `config.local.release.php` in the directory `conf`and try to add it and call it with a `require 'config.local.release.php'`

Reply:
resolve with #73